openseeds/cohort-procedures-and-templates/microgrants.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: page
+title: Microgrants
+image: https://images.unsplash.com/photo-1579621970588-a35d0e7ab9b6
+photos:
+  name: Micheile Henderson
+  license: CC-BY
+  url: https://images.unsplash.com/photo-1579621970588-a35d0e7ab9b6
+---
+
+# OLS Microgrant Policy and processes
+
+## Applicant workflow
+
+1. **Requesting a microgrant:** Accepted OLS-participants are asked if they need a microgrant as part of OLS registration. To request a microgrant at other times, email microgrants@we-are-ols.org with details of your request. 
```

Comment:
Please use a Jekyll variable for the email (maybe just for the 2nd part)

Reply:
We currently don't have a variable that stores just `we-are-ols.org`...

Reply:
Feel free to create one :)